hail/python/hail/utils/struct.py

```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
         -------
         :class:`.Struct`
             Struct without certain fields.
+
+        Examples
+        --------
+
+        Define a Struct `s`
+
+         >>> s = hl.Struct(food=8, fruit=5, bar=2, apple=10)
+
+        Drop one field from `s`
+
+         >>> s.drop(bar=2)
+         Struct(food=8, fruit=5, apple=10)
+
+        Drop two fields from `s`
+
+         >>> s.annotate(food=8, fruit=5)
```

Comment:
looks like this is an annotation instead of a drop!

Reply:
Fixed